docs/started/main_components/daemons.md

```diff
@@ -10,47 +10,47 @@ and then write some data back into the DB.
-rucio-oauth-manager | Auth/Authz | Deletes expired access tokens (in case there is a valid refresh token, expired access tokens will be kept until refresh_token expires as well.) and deletion of expired OAuth session parameters | [Details](bin/rucio-oauth-manager.md)
-rucio-reaper | Deletion | Deletes replicas that don't have locks anymore, i.e. they have a tombstone set | [Details](bin/rucio-reaper.md)
-rucio-suspicious-replica-recoverer | Replica | Declares suspicious replicas as bad if they are found available on other RSEs, so necromancer will work on them | [Details](bin/rucio-replica-recoverer.md)
-rucio-rse-decommissioner | Deletion | Decommissions an RSE. The actions to perform are specified in decommissioning profiles (delete all data, move replicas, etc ...)  | [Details](bin/rucio-rse-decommissioner.md)
-rucio-storage-consistency-actions | Consistency | Applies corrective actions as a result of a consistency check on an RSE | [Details](bin/rucio-storage-consistency-actions.md)
-rucio-transmogrifier | Rule | Creates replication rules for DIDs matching a subscription | [Details](bin/rucio-transmogrifier.md)
-rucio-undertaker | Deletion | Manages expired DIDs, deleting them (does not delete replicas)  | [Details](bin/rucio-undertaker.md)
+Name | Domain|  Purpose |  Batched? | Threaded? | Multi-VO? | Details
+-----------| ----------------- | -------| ----- | ----- | ----- | ----- |
+rucio-abacus-account | Accounting | Account usage | ❌ | ✅ | ❌ |  [Details](bin/rucio-abacus-account.md)
+rucio-abacus-collection-replica | Accounting |  Updates collection replicas | ❌ | ✅ | ❌ |  [Details](bin/rucio-abacus-collection-replica.md)
+rucio-abacus-rse | Accounting | Updates RSE counters | ❌ | ✅ | ❌ |  [Details](bin/rucio-abacus-rse.md)
+rucio-atropos | Replica | End the life of the rules according to the Lifetime Model | ❌ | ✅ | ❌ |  [Details](bin/rucio-atropos.md)
+rucio-auditor | Replica | Find inconsistencies on storage, for example, dark data discovery | ❌ | ❌ | ❌ |  [Details](bin/rucio-auditor.md)
+rucio-automatix | Replica | Used for testing: injects random data in RSEs to check liveness | ❌ | ✅  | ❌ |  [Details](bin/rucio-automatix.md)
+rucio-bb8 | Replica | Rebalance data across RSEs | ❌ | ✅ | ❌ |  [Details](bin/rucio-bb8.md)
+rucio-cache-client | Replica | Populates information of replicas on volatile storage  | ❌ | ❌ | ❌ |  [Details](bin/rucio-cache-client.md)
+rucio-cache-consumer | Replica | Adds and deletes cache replicas to the Rucio catalog | ❌ | ✅ | ❌ |  [Details](bin/rucio-cache-consumer.md)
+rucio-conveyor-finisher | Transfer | Updates Rucio internal state after the file transfer has finished | ✅  | ✅ | ❌ |  [Details](bin/rucio-conveyor-finisher.md)
+rucio-conveyor-poller | Transfer | Polls updates from the transfer tool to check the transfer state | ❌ | ✅ | ❌ |  [Details](bin/rucio-conveyor-poller.md)
```

Comment:
this one seems to be batched too https://rucio.github.io/documentation/bin/rucio-conveyor-finisher

Reply:
LGTM, just this one that's still unresolved